mpas_analysis/test/test_climatology.py

```diff
@@ -110,13 +110,17 @@ def test_get_mpas_climatology_file_names(self):
         config = self.setup_config()
         fieldName = 'sst'
         monthNames = 'JFM'
-        (climatologyFileName, regriddedFileName) = \
+        (climatologyFileName, climatologyPrefix, regriddedFileName) = \
             climatology.get_mpas_climatology_file_names(config, fieldName,
                                                         monthNames)
         expectedClimatologyFileName = '{}/clim/mpas/sst_QU240_JFM_' \
                                       'years0002-0002.nc'.format(self.test_dir)
         self.assertEqual(climatologyFileName, expectedClimatologyFileName)
 
+        expectedClimatologyPrefix = '{}/clim/mpas/sst_QU240_' \
+                                    'JFM'.format(self.test_dir)
+        self.assertEqual(climatologyPrefix, expectedClimatologyPrefix)
+
```

Comment:
This would be great.  The goal, though, was to get this code working and fast.  I spent the time I had getting it to work on real cases we have to solve.  Unit tests are definitely a good idea but they typically takes significantly longer to set up and debug, especially for these more complex problems like climatologies, as it does to incorporate them into the real analysis tasks.

It's 2 am and I've already put half my weekend into this PR that I didn't really have to spare. I'll see if I can get to this tomorrow...